---
# 控制器配置
# Publish all joint states -----------------------------------
joint_states_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

# Position Controllers ---------------------------------------
fr_steer_left_joint_controller:
    type: effort_controllers/JointPositionController
    joint: fr_steer_left_joint
    pid: {p: 20.0, i: 1.0, d: 1}

fr_steer_right_joint_controller:
    type: effort_controllers/JointPositionController
    joint: fr_steer_right_joint
    pid: {p: 20.0, i: 1.0, d: 1}

#Velocity Controllers ---------------------------------------
fr_left_joint_controller:
    type: effort_controllers/JointVelocityController
    joint: fr_left_joint
    pid: {p: 12.0, i: 2, d: 0.03, i_clamp: 20}

fr_right_joint_controller:
    type: effort_controllers/JointVelocityController
    joint: fr_right_joint
    pid: {p: 12.0, i: 2, d: 0.03, i_clamp: 20}

re_left_joint_controller:
    type: effort_controllers/JointVelocityController
    joint: re_left_joint
    pid: {p: 12.0, i: 2, d: 0.03, i_clamp: 20}

re_right_joint_controller:
    type: effort_controllers/JointVelocityController
    joint: re_right_joint
    pid: {p: 12.0, i: 2, d: 0.03, i_clamp: 20}